### ISSUE TITLE
Platform inconsistencies in GSUB5/GPOS7 lookup check rationale

### ISSUE THREAD

**Issue by chrissimpkins** [googlefonts/fontbakery#3689]
Check: `com.google.fonts/check/gsub5_gpos7`
Added: https://github.com/googlefonts/fontbakery/pull/3644

The rationale for the GSUB5/GPOS7 check states:

```
 Ensure no GSUB5/GPOS7 lookups are present.
    with build/font.ttf

      Rationale:                                                                
      Versions of fonttools >=4.14.0 (19 August 2020) perform an optimisation on
      chained contextual lookups, expressing GSUB6 as GSUB5 and GPOS8 and GPOS7 
      where possible (when there are no suffixes/prefixes for all rules in the  
      lookup).                                                                  
                                                                                
      However, makeotf has never generated these lookup types and they are rare 
      in practice. Perhaps before of this, Mac's CoreText shaper does not       
      correctly interpret GPOS7, and certain versions of Windows do not         
      correctly interpret GSUB5, meaning that these lookups will be ignored by  
      the shaper, and fonts containing these lookups will have unintended       
      positioning and substitution errors.                                      
                                                                                
      To fix this warning, rebuild the font with a recent version of fonttools. 

     WARN Font contains a GPOS7 lookup which is not processed by macOS          
          [code: has-gpos7]                                                     

     WARN Font contains a GSUB5 lookup which is not processed by macOS          
          [code: has-gsub5]                                                     


    Result: WARN
```

The body of the rationale indicates that the GPOS7 issue is macOS-specific and GSUB5 issue is Windows-specific.  The end of the report includes two WARN fields that indicate GPOS7 **and** GSUB5 are macOS-specific issues.  If the body of the rationale is valid, I think that it is only necessary to change:

```
WARN Font contains a GSUB5 lookup which is not processed by macOS          
          [code: has-gsub5] 
```

to:

```
WARN Font contains a GSUB5 lookup which is not processed by some versions of Windows          
          [code: has-gsub5] 
```

There also appears to be a minor typo in line 2 of paragraph 2.  "Perhaps before of this" ==> "Perhaps because of this"?

**Comment by khaledhosny** [comment]
There is no proof about GSUB5 being an issue on Windows, it was just speculation IIRC.

**Comment by simoncozens** [comment]
Yes, I'm sorry, that was my fault. I saw that Yanone was having problems with a font which was generating GSUB5 lookups, and I had just been working on this issue and jumped to conclusions. GPOS 7 is definitely a problem, though.

**Comment by felipesanches** [comment]
This was addressed by @simoncozens at #3690
